controller/WebController.php

```diff
@@ -213,18 +213,17 @@ public function invokeFeedbackForm($request)
         }
         $feedbackName = $request->getQueryParamPOST('name');
         $feedbackEmail = $request->getQueryParamPOST('email');
+        $msgSubject = $this->model->getConfig()->getServiceName() . ": ";
```

Comment:
Also, this is really nitpicking but...since the formatting of the message body is done in the sendFeedback method, I think it would be more consistent to also format the subject line within sendFeedback, not here. So just pass the `msgsubject` POST variable as it is to sendFeedback, then add the prefix within sendFeedback - where the subject line used to be formatted before this change.

Reply:
Good ideas - Done!